proto/sourcehub/orbis/events.proto

```
@@ -45,3 +45,22 @@ message EventRingDeleted {
   string ring_id = 1;
   string reason = 2;
 }
+
+message EventRingUpgradeScheduled {
+  string ring_id = 1;
+  uint64 current_version = 2;
+  uint64 next_version = 3;
+  uint64 activation_time = 4;
+}
```

Buf (1.70.0)

[error] Message "EventRingUpgradeScheduled" should have a non-empty comment for documentation.

(COMMENT_MESSAGE)